pkg/test/step_test.go

```diff
@@ -53,16 +54,16 @@ func TestStepClean(t *testing.T) {
 // Each test provides a path to a set of test steps and their rendered result.
 func TestStepCreate(t *testing.T) {
 
-	pod := testutils.NewPod("hello", "")
+	pod := testutils.NewPod("hello", "default")
```

Comment:
another case where a test started to fail without an explicit `default` namespace.

Reply:
:thinking: I feel like this behavior as tested is wrong. It should only be setting the namespace if it isn't set. Do you know what changed?

Reply:
So, for anybody who is interested: after some researching and debugging, I believe it was [this change in go-client testing fixture](https://github.com/kubernetes/kubernetes/commit/d32c76fc03381784516c47cb1bf62ef932189afa#diff-1fedd5b881e8fbda5aaefa9bd74b359bR285) that made this test fail. The commit description says it all.
As for the default client, this is also [explicitly forbidden](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/rest/request.go#L789). One mistery solved.